tests/Feature/PageHomeTest.php

```diff
@@ -20,7 +20,7 @@ public function it_shows_given_streams_on_home_page(): void
         Stream::factory()->create(['title' => 'Stream #3', 'scheduled_start_time' => Carbon::now()->addDays(3), 'youtube_id' => '123456']);
 
         // Act & Assert
-        $this->get('/?timezone=Europe/Vienna')
+        $this->get('/?timezone=UTC')
```

Comment:
you can remove the query param from this test, this was used in the old solution.